internal/config/config.go

```diff
@@ -151,7 +151,7 @@ func ReadConfig(path string, readFn ReadFileFn) (*Config, error) {
 		return nil, err
 	}
 	config := &Config{}
-	if err := yaml.UnmarshalStrict(data, config); err != nil {
+	if err := yaml.Unmarshal(data, config); err != nil {
```

Comment:
We would like to preserve strict unmarshalling behaviour in yaml.3 as well.
This is possible via `Decoder.KnownFields`, can you please reflect on this ?

From [API v3 of the yaml package for Go is available](https://ubuntu.com/blog/api-v3-of-the-yaml-package-for-go-is-available):
> The general “strict” term is gone from the API, and has been replaced by the more specific Decoder.KnownFields method which forces the decoder to ensure that any mapping keys being decoded from the YAML text are present in the struct being decoded into.

Reply:
@mikouaj : Thanks  ,  addressed the review comments